src/activity_actor.cpp

```diff
@@ -6154,6 +6154,11 @@ void firstaid_activity_actor::finish( player_activity &act, Character &who )
     static const std::string iuse_name_string( "heal" );
 
     item_location it = act.targets.front();
+    if( it.get_item() == nullptr ) {
```

Comment:
Use the `item_location`'s bool operator instead

```suggestion
    if( it ) {
```
and merge this check into the `if` below.

There's not much point to this though. An activity that fails with a debug message for no discernible reason is not much better than a crash from the player's perspective.

Reply:
Thank you for reviewing! How do you suggest I merge with the other `if`? I didn't see a clear way without calling `get_usable_item` twice.